pkg/apiserver/experiment/experiment.go

```diff
@@ -885,6 +890,26 @@ func (s *Service) deleteExperiment(c *gin.Context) {
 		return
 	}
 
+	if force == "true" {
+		if chaosMeta, ok = chaosKind.Chaos.(metav1.Object); !ok {
+			c.Status(http.StatusInternalServerError)
+			_ = c.Error(utils.ErrInternalServer.WrapWithNoMessage(fmt.Errorf("failed to get chaos meta information")))
+			return
+		}
+
+		annotations := chaosMeta.GetAnnotations()
+		if annotations == nil {
+			annotations = make(map[string]string)
+		}
+		annotations[common.AnnotationCleanFinalizer] = common.AnnotationCleanFinalizerForced
+		chaosMeta.SetAnnotations(annotations)
+		if err := s.kubeCli.Update(context.Background(), chaosKind.Chaos); err != nil {
+			c.Status(http.StatusInternalServerError)
+			_ = c.Error(utils.ErrInternalServer.WrapWithNoMessage(fmt.Errorf("update chaos annotation error")))
```

Comment:
A small nit here. I am wondering if we can make this error message more clear, since we are doing a force delete via adding an annotation, the real deletion is performed at the controller side.

But users might feel strange when they see `update chaos  annotation error` when they think they just want to delete a chaos.

Reply:
Okay, I will modify this information.